cli/query/query.go

```diff
@@ -113,3 +127,65 @@ func dumpApplicationLog(ctx *cli.Context, res *result.ApplicationLog, tx *result
 	_ = tw.Flush()
 	fmt.Fprint(ctx.App.Writer, buf.String())
 }
+
+func queryCandidates(ctx *cli.Context) error {
+	var err error
+
+	gctx, cancel := options.GetTimeoutContext(ctx)
+	defer cancel()
+
+	c, err := options.GetRPCClient(gctx, ctx)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	vals, err := c.GetNextBlockValidators()
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	comm, err := c.GetCommittee()
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	sort.Slice(vals, func(i, j int) bool {
+		if vals[i].Active != vals[j].Active {
+			return vals[i].Active
+		}
+		if vals[i].Votes != vals[j].Votes {
+			return vals[i].Votes > vals[j].Votes
+		}
+		return vals[i].PublicKey.Cmp(&vals[j].PublicKey) == -1
+	})
+	buf := bytes.NewBuffer(nil)
+	tw := tabwriter.NewWriter(buf, 0, 2, 2, ' ', 0)
+	_, _ = tw.Write([]byte("Key\tVotes\tCommittee\tConsensus\n"))
+	for _, val := range vals {
+		_, _ = tw.Write([]byte(fmt.Sprintf("%s\t%d\t%t\t%t\n", hex.EncodeToString(val.PublicKey.Bytes()), val.Votes, comm.Contains(&val.PublicKey), val.Active)))
```

Comment:
Is it possible that candidate does not belong to committee?

Reply:
We can have like 50 candidates with 21 committee members.